xml/core_registers.xml

```diff
@@ -156,13 +156,16 @@
         </field>
         <field name="stoptime" bits="9" access="WARL" reset="Preset">
             <value v="0" name="normal">
-            Increment \Rtime as usual.
+            \Rtime continues to reflect \Rmtime.
             </value>
 
             <value v="1" name="freeze">
-            Don't increment \Rtime while in Debug Mode.  If all harts
-            have \FcsrDcsrStoptime=1 and are in Debug Mode then \Rmtime
-            is also allowed to stop incrementing.
+            \Rtime is frozen at the time that Debug Mode was entered. When
+            leaving Debug Mode, \Rtime will snap forward to reflect the latest
+            value of \Rmtime again.
```

Comment:
I don't like the term "snap forward" because it seems like slang.  Thinking some more, it might not even be technically true.  If mtime stops counting (due to the sentence below) then it actually won't snap forward at all.  It'll just continue to increment when mtime starts incrementing again.

```suggestion
            \Rtime is frozen at the time that Debug Mode was entered. When
            leaving Debug Mode, \Rtime will reflect the latest
            value of \Rmtime again.
```